### DISCUSSION TITLE
如果最後一批測試成功，就build 出來吧

### DISCUSSION THREAD

**Discussion by bestian** [3dw/auto20-next#204] [General]
@leechiuhui 目前這一批大型的修改好像完成了，測試的部分您比我來得仔細

如果最後一批測試成功，就build 出來吧。也是一個蠻好的里程碑。謝謝合作開發🙏

**Comment by leechiuhui** [comment]
老師您好：今天外出較久，有些疲憊。剛才進行了約十分鐘的簡單測試，發現仍存在一些問題，但具體描述有些困難。懇請給我更多時間處理，感謝您的理解！🙏

**Reply by bestian** [reply]
瞭解，休息夠再說。
對不起讓你有被push的感覺~

**Reply by bestian** [reply]
@leechiuhui 
我只是自己不敢build...怕測得不夠仔細。不是要催你

**Reply by leechiuhui** [reply]
Teacher,I know ,too,😁上面回覆我用ChatGPT 潤稿潤的太禮貌了啦,我只是想說：今天在倫敦出外探險太累了，想測試但有點難描述問題，加上明天又要空中移動到新加坡了，再加上時差怕耽誤這裡時間….還是會趁頭腦清醒時強力測試的😁 ps:剛剛還弄錯帳號回覆，「sunshinejohnny」這個帳號是我用nus的學生身份申請GitHub看看的⋯🫠

**Reply by leechiuhui** [reply]
@bestian 老師如果您測試沒問題就直接build吧！畢竟auto20-next是您親生的⋯⋯您測絕對沒問題的啦，就算有問題你也可以馬上救起來😁🌹

**Reply by bestian** [reply]
馬上救起來是不一定, 也要時間許可. 不過該build還是要build, 我最後修了幾個小bug, 就build了

**Reply by bestian** [reply]
@leechiuhui 
此外，因為我發現自己最近也有點過度投入, 可能是因為要交給理監事會看新版自學2.0的關係. 不過現階段已經很ok了. 我想我們都放鬆到10月再繼續吧. ~

**Reply by leechiuhui** [reply]
好喔～😁🌻
